.github/workflows/release.yml

```diff
@@ -20,7 +20,6 @@ jobs:
   Deploy:
     name: Deploy
     runs-on: ubuntu-latest
-    needs: [Build]
```

Comment:
I removed the dependency onto build for now as we currently do all these checks locally (and artifacts published) and sometimes CI can be flaky, but we always want to make sure we publish the latest docs/release etc